llama/s3.py

```diff
@@ -3,7 +3,7 @@
 
 
 class S3:
-    """An S3 class that provides a generic boto3 s3 client for interacting with S3
+    """A S3 class that provides a generic boto3 s3 client for interacting with S3
```

Comment:
Doesn't really matter, but "an" is technically the correct article here and shouldn't be changed (English is a ridiculous language) :)